plugins/modules/tanzu_mission_control_secret.py

```diff
@@ -0,0 +1,778 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2025, Massimo Gengarelli (massimo.gengarelli@proton.me)
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+module: tanzu_mission_control_secret
+short_description: Manages Cluster, Cluster Group Secrets and SecretExports in Tanzu Mission Control
+version_added: 11.0.0
+description:
+  - Create and revokes Cluster and Cluster Group Secrets in Tanzu Mission Control for a given Cluster.
+  - Creates and removes SecretExports for the given secret in Tanzu Mission Control.
+author:
+  - Massimo Gengarelli (@massix)
+requirements: [requests]
+notes:
+  - To obtain an C(api_token), login to Tanzu Mission Control, click on your user, then I(Settings) and then I(Generate an
```

Comment:
Since `api_token` is an option, you should use `O(...)`:
```suggestion
  - To obtain an O(api_token), login to Tanzu Mission Control, click on your user, then I(Settings) and then I(Generate an
```